pennylane/math/__init__.py

```diff
@@ -69,6 +69,7 @@
 from .is_independent import is_independent
 
 sum = ar.numpy.sum
+add = ar.numpy.add
```

Comment:
I know this is kind of hacky but this should help:

```
def add(*args, **kwargs):
    try:
        return ar.numpy.add(*args, **kwargs)
    except TypeError:
        # catch arg1 = torch, arg2=numpy error
        # works fine with opposite order
        return ar.numpy.add(args[1], args[0], *args[2:], **kwargs)
```

Reply:
Awesome! I was just about to push a very similar change: 

```
def add(tensor1, tensor2, **kwargs):
    tensors = [tensor1, tensor2]
    interfs = [get_interface(tensor) for tensor in tensors]

    if "numpy" in interfs and "torch" in interfs:
        numpy_idx = interfs.index("numpy")
        tensors[numpy_idx] = ar.numpy.asarray(tensors[numpy_idx], like="torch")

    return ar.numpy.add(tensors[0], tensors[1], **kwargs)
```

I like your solution because its much more concise, but do you think this would be better for code readability?